internal/docstore/firedocstore/fs.go

```diff
@@ -93,17 +93,23 @@ type Options struct {
 	MaxOutstandingActionRPCs int
 }
 
+// CollectionResourceID constructs a resource ID for a collection from the project ID and the collection path.
+// See the OpenCollection example for use.
+func CollectionResourceID(projectID, collPath string) string {
+	return fmt.Sprintf("projects/%s/databases/(default)/documents/%s", projectID, collPath)
+}
+
 // OpenCollection creates a *docstore.Collection representing a Firestore collection.
 //
-// collPath is the path to the collection, starting from a root collection. It may
-// refer to a top-level collection, like "States", or it may be a path to a nested
+// collResourceID must be of the form "project/<projectID>/databases/(default)/documents/<collPath>".
```

Comment:
If you can find a link documenting this form, please add. Especially if it includes stuff like advice on how to copy/paste from the UI.

Reply:
Done, as best I could.